src/main/java/de/tudarmstadt/ukp/wikipedia/parser/mediawiki/ModularParser.java

```diff
@@ -1074,7 +1074,8 @@ private void convertGalleriesToImages(SpanManager sm, List<Span> tagSpans)
                     for (String imageParagraph : tokenize(sm, startSpan.getEnd(), endSpan
                         .getStart(), lineSeparator))
                     {
-                        String[] splitImageParagraph = imageParagraph.split("\\|");
+						String[] splitImageParagraph = imageParagraph.split("\\|", -1);
+
```

Comment:
This line has tabs, but the other ones have spaces.
https://www.dropbox.com/s/s2eve5pcrtmdkn2/Screenshot%202015-08-26%2010.13.14.png?dl=0